static/js/around-map.js

```diff
@@ -70,7 +81,7 @@ map.on('click', event => {
 // functions
 function currentMarker (currentLatLng) {
   const popup = new mapboxgl.Popup({offset: [10, 0]})
-    .setText('Your current location')
+    .setHTML('<p>Your current location</p>')
```

Comment:
I would just say "<p>My current location</p>" instead, as we are using My/I/me on the project.